### PR TITLE
Ec graph issue 31

### LINKED ISSUE #31: Graph: Display Risk vs. Group size data
### User Story
As a user, I want to view a graph on the data, so that I can see the risk vs. group size data displayed

### Additional Context
Screenshots, links to resources or discussions, anything else that might help us understand more about this issue

### Acceptance Criteria
- [ ] Get the graph to display

### PR DESCRIPTION
**Pull request by chonger878**
## Summary
Closes #31

A graph of the relationship between group size and risk is displayed.  Data points are displayed on the graph.

If there are many changes, make a list of them.
*Hashmaps of state and country level risks were created by Larissa
*Code cleaned up by Larissa to display risks at state, county, and country level

## Checklist
- [ ] New tests added to account for changes in this MR
- [ ] All tests passing locally
- [ ] All tests passing on CircleCI
- [x ] Assigned at least 2 reviewers

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,9 +3,12 @@ buildscript {
     repositories {
         google()
         jcenter()
+
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.0.1"
+
```
Comment (unicornsleuth) [inline build.gradle]: revert this file change

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Thu Oct 22 17:47:05 PDT 2020
+#Sun Nov 15 20:08:13 PST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.4.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```
Comment (unicornsleuth) [inline gradle/wrapper/gradle-wrapper.properties]: revert this file change

**Review by unicornsleuth — CHANGES_REQUESTED** [review]
Looks great! Just change some of the build files and you're good to go!

File: build.gradle (hunk 1 above)
Comment (chonger878) [reply build.gradle]: Done and done.

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (chonger878) [reply gradle/wrapper/gradle-wrapper.properties]: done and Done.

**Review by dansjack — APPROVED** [review]
Looks great! It still looks good when rotating the device too

**Review by unicornsleuth — APPROVED** [review]
Works as expected, and thanks for making the changes I requested!

**Pull request merged**